docker-for-windows/install.md

```diff
@@ -1,52 +1,51 @@
 ---
 description: How to install Docker Desktop for Windows
 keywords: windows, beta, edge, alpha, install, download
-title: Install Docker Desktop for Windows
+title: Install Docker Desktop on Windows
 ---
 
-Docker Desktop for Windows is the [Community Edition
-(CE)](https://www.docker.com/community-edition) of Docker for Microsoft Windows.
-To download Docker Desktop for Windows, head to Docker Hub.
+Docker Desktop for Windows is the [Community](https://www.docker.com/community-edition) version of Docker for Microsoft Windows.
+You can download Docker Desktop for Windows from Docker Hub.
 
 [Download from Docker
 Hub](https://hub.docker.com/?overlay=onboarding){:
 .button .outline-btn}
 
-##  What to know before you install
-
-* **README FIRST for Docker Toolbox and Docker Machine users**: Docker Desktop for
-  Windows requires Microsoft Hyper-V to run. The Docker Desktop for Windows installer
-  enables Hyper-V for you, if needed, and restarts your machine. After Hyper-V is
-  enabled, VirtualBox no longer works, but any VirtualBox VM images remain.
-  VirtualBox VMs created with `docker-machine` (including the `default` one
-  typically created during Toolbox install) no longer start. These VMs cannot be
-  used side-by-side with Docker Desktop for Windows. However, you can still use
-  `docker-machine` to manage remote VMs.
-
-* **System Requirements**:
-  - Windows 10 64bit: Pro, Enterprise or Education (Build 15063 or later).
-  - Virtualization is enabled in BIOS. Typically, virtualization is enabled by
-    default. This is different from having Hyper-V enabled. For more detail see
-    [Virtualization must be
-    enabled](troubleshoot.md#virtualization-must-be-enabled) in Troubleshooting.
-  - CPU SLAT-capable feature.
-  - At least 4GB of RAM.
-
->  **Note**: If your system does not meet the requirements to run Docker Desktop for
-> Windows, you can install [Docker Toolbox](/toolbox/overview.md), which uses
-> Oracle Virtual Box instead of Hyper-V.
-
-* **What the Docker Desktop for Windows install includes**: The installation provides
-  [Docker Engine](/engine/userguide/), Docker CLI client, [Docker
-  Compose](/compose/overview.md), [Docker Machine](/machine/overview.md), and
-  [Kitematic](/kitematic/userguide.md).
-* Containers and images created with Docker Desktop for Windows are shared between all
-  user accounts on machines where it is installed. This is because all Windows
-  accounts use the same VM to build and run containers.
-* Nested virtualization scenarios, such as running Docker Desktop for Windows on a
-  VMWare or Parallels instance might work, but there are no guarantees. For
-  more information, see [Running Docker Desktop for Windows in nested virtualization
-  scenarios](troubleshoot.md#running-docker-desktop-for-windows-in-nested-virtualization-scenarios)
+## What to know before you install
+
+### System Requirements
+
+  - Windows 10 64-bit: Pro, Enterprise or Education (Build 15063 or later).
```

Comment:
s/Pro, Enterprise or Education/Pro, Enterprise, or Education

Reply:
Fixed in commit e055fb79.